packages/guides/src/Nodes/BreadCrumbNode.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Nodes;
+
+/** @extends AbstractNode<string> */
+class BreadCrumbNode extends AbstractNode
+{
+    public function __construct(string $value = '')
+    {
+        $this->value = $value;
```

Comment:
Please call the parent constructor, this prevents issues when we change the parent class.

Reply:
Calling the parent constructor on a class that has no constuctor is not possible...